tests/_benderjs/ckeditor/static/tools.js

```diff
@@ -1278,6 +1278,32 @@
 				mouseEvent.initMouseEvent( type, true, true, window, 0, 0, 0, 80, 20, false, false, false, false, button, null );
 				element.dispatchEvent( mouseEvent );
 			}
+		},
+
+		/**
+		 * Fires element event handler attribute e.g.
+		 * ```html <button onkeydown="return customFn( event )">x</button>```
+		 *
+		 * @param {CKEDITOR.dom.element/HTMLElement} element Element with attached event handler attribute.
+		 * @param {String} eventName Event handler attribute name.
+		 * @param {Object} evt Event payload.
+		*/
+		fireElementEventHandler: function( element, eventName, evt ) {
+			if ( element.$ ) {
+				element = element.$;
+			}
+
+			if ( CKEDITOR.env.ie && CKEDITOR.env.version < 9 ) {
+				var nativeEvent = CKEDITOR.document.$.createEventObject();
+
+				for ( var key in evt ) {
+					nativeEvent[ key ] = evt[ key ];
+				}
+
+				element.fireEvent( eventName, nativeEvent );
+			} else {
+				element[ eventName ]( evt );
+			}
```

Comment:
We don't need that helper function, as there is already [element#fireEventHandler](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_dom_element.html#method-fireEventHandler) method that covers this polyfill (that helper was moved to core codebase some time ago).